src/rebar_utils.erl

```diff
@@ -158,11 +158,21 @@ sh(Command0, Options0) ->
     ?DEBUG("Port Cmd: ~s\nPort Opts: ~p\n", [Command, PortSettings]),
     Port = open_port({spawn, Command}, PortSettings),
 
-    case sh_loop(Port, OutputHandler, []) of
-        {ok, _Output} = Ok ->
-            Ok;
-        {error, {_Rc, _Output}=Err} ->
-            ErrorHandler(Command, Err)
+    case os:type() of
+        {win32, _} ->
```

Comment:
Hmm.. true.

I see that it uses it for git, hg and so..

Need to go to bed now. But if you want you can remove the sh_loop_win32 and add the checks in xcopy_win32 instead and the mv function?